tests/test_port.py

```diff
@@ -70,6 +70,29 @@ def test_PortNotification(self, dvs, testlog):
 
         assert oper_status == "up"
 
+    def test_PortFecForce(self, dvs, testlog):
+        db = swsscommon.DBConnector(0, dvs.redis_sock, 0)
+        adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
+
+        tbl = swsscommon.Table(db, "PORT_TABLE")
+        ptbl = swsscommon.ProducerStateTable(db, "PORT_TABLE")
+        atbl = swsscommon.Table(adb, "ASIC_STATE:SAI_OBJECT_TYPE_PORT")
+
+        # set fec
+        fvs = swsscommon.FieldValuePairs([("fec","none")])
+        ptbl.set("Ethernet0", fvs)
+
+        time.sleep(1)
```

Comment:
i think on another PR stepan mentioned that tests moved to wait instead of sleep. can you please check?

should we test it on different type of ports? breakout port and non breakout port?
should you test different fec options? only only none?
how this test cover the new push flow?